slices/apache2.yaml

```diff
@@ -0,0 +1,316 @@
+package: apache2
+
+essential:
+  - apache2_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2-bin_bins
+      - apache2_configs
+      - apache2-data_data
+      - apache2-utils_bins
+      - media-types_data
```

Comment:
We should add `coreutils_bins`, as it is required for `apache2ctl` (which uses mkdir, chmod & chown utilities).

Reply:
And we should also add `base-files_bin`.